benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -262,6 +264,26 @@ def get_all_sorting_searching_operators():
     return sort_search_mx_operators
 
 
+def get_all_rearrange_operators():
+    """Gets all array rearrange operators registered with MXNet.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    rearrange_ops = ['transpose','swapaxes','flip','depth_to_space','space_to_depth']
```

Comment:
This should be a set if repeatedly testing for in.

Could we refactor all these get_ functions and make some kind of yaml file where we add tags to the list of operators, I think a declarative thing would be much easier to maintain than all this logic which can be factored out just to select some ops based on the names.

Let me know if what I'm suggesting is not clear.

Reply:
Thanks for clarifying this over chat. Seems there's slightly different logics in some of these functions.